Lib/test/test_argparse.py

```diff
@@ -3901,7 +3901,7 @@ class TestHelpVariableExpansion(HelpTestCase):
     parser_signature = Sig(prog='PROG')
     argument_signatures = [
         Sig('-x', type=int,
-            help='x %(prog)s %(default)s %(type)s %%'),
+            help='x %(prog)s %(default)s %(type)s %'),
```

Comment:
This is a breaking change.

Reply:
I have reverted this change and added a line to replace the escaped `%%` in the `_expand_help` method so now it should be fully compatible with the old string formatting too.